tests/Resources/ti.ui.webview.test.js

```diff
@@ -728,4 +730,75 @@ describe('Titanium.UI.WebView', function () {
 		});
 		should(webView).have.a.property('progress').which.is.a.Number(); // should default to 0 until we start loading the page.
 	});
+
+	it.ios('#findString', function (finish) {
+		if (OS_VERSION_MAJOR < 14) {
+			return finish();
+		}
+		win = Ti.UI.createWindow();
+		const webView = Ti.UI.createWebView({
+			url: 'https://www.google.com'
+		});
+
+		webView.addEventListener('load', function () {
+			webView.findString('google', function (e) {
```

Comment:
add another test with the search options passed in? (i.e. case sensitive and search by bad case). Test "negative" case: search for something you know you won't find.

Also, this API leaves a bit to be desired. The options for wrapping/searching backwards don't seem very useful if we only get a boolean whether the search was found (rather than say also a position or something). But maybe it matters because it updates some UI state to highlight the result(s)?

Reply:
In API doc,  it is mentioned that  "A match found by the search is selected and the page is scrolled to reveal the selection. The completion handler is called after the search completes". But it doesn't scroll and highlight in native as well. Probably beta issue.